### ISSUE TITLE
Suggestion: Rare price prediction

### ISSUE THREAD

**Issue by TheOne320** [Kyusung4698/PoE-Overlay#178]
I know many people frown on this, but I like to use poeprices.info to get a quick estimate for rare items and if something jumps out, I check it manually. Could you please implement price prediction using poeprices.info? If you do, please also display the weightings that contribute to the predicted price. You could make it toggle-able for people who hate it.

![image](https://user-images.githubusercontent.com/7705568/73609370-52eb8380-45cd-11ea-92d9-34249fe04d7d.png)

**Comment by 766b** [comment]
If I might add to this, it would be nice to have a feature pricing whole dump tab then highlighting result in-game by drawing a border around items with pricing info and extended details upon hover. This will require using stash tab API, and poeprices API (if they have one, technically response JSON is hardcoded into HTML page, but using that would be in bad taste without permission.)

It can be useful to find items that might be worth something if the ability to filter results is added (for example only show items worth more than 25 chaos).

Another way would be to submit data from stash tab then open a browser page with results ingame. The drawback is that poeprices has abysmal UI for results that are hard to read.

**Comment by JusKillmeQik** [comment]
> If I might add to this, it would be nice to have a feature pricing whole dump tab then highlighting result in-game by drawing a border around items with pricing info and extended details upon hover. This will require using stash tab API, and poeprices API (if they have one, technically response JSON is hardcoded into HTML page, but using that would be in bad taste without permission.)
> 
> It can be useful to find items that might be worth something if the ability to filter results is added (for example only show items worth more than 25 chaos).
> 
> Another way would be to submit data from stash tab then open a browser page with results ingame. The drawback is that poeprices has abysmal UI for results that are hard to read.

PoePrices already does this: https://www.poeprices.info/#priceastash

**Comment by JusKillmeQik** [comment]
To the main feature request of adding the pricing info, I'd like to add more details to make this happen quickly. I know there are a lot of Issues right now with league launch, but this is the last feature I need to make the switch completely to this tool, so I'll do what I can to help push it along.

All you need to do is take the exact item text:

```
Rarity: Rare
Armageddon Dash
Crusader Boots
--------
Quality: +20% (augmented)
Armour: 174 (augmented)
Energy Shield: 42 (augmented)
--------
Requirements:
Level: 70
Str: 62
Dex: 111
Int: 155
--------
Sockets: G-B-B-B 
--------
Item Level: 86
--------
+39 to Strength
+24 to Armour
+11 to maximum Energy Shield
+45 to maximum Mana
35% increased Movement Speed
+19% to Fire and Cold Resistances (crafted)
```

And Base64 Encode it:

UmFyaXR5OiBSYXJlCkFybWFnZWRkb24gRGFzaApDcnVzYWRlciBCb290cwotLS0tLS0tLQpRdWFsaXR5OiArMjAlIChhdWdtZW50ZWQpCkFybW91cjogMTc0IChhdWdt
ZW50ZWQpCkVuZXJneSBTaGllbGQ6IDQyIChhdWdtZW50ZWQpCi0tLS0tLS0tClJlcXVpcmVtZW50czoKTGV2ZWw6IDcwClN0cjogNjIKRGV4OiAxMTEKSW50OiAxNTUK
LS0tLS0tLS0KU29ja2V0czogRy1CLUItQiAKLS0tLS0tLS0KSXRlbSBMZXZlbDogODYKLS0tLS0tLS0KKzM5IHRvIFN0cmVuZ3RoCisyNCB0byBBcm1vdXIKKzExIHRv
IG1heGltdW0gRW5lcmd5IFNoaWVsZAorNDUgdG8gbWF4aW11bSBNYW5hCjM1JSBpbmNyZWFzZWQgTW92ZW1lbnQgU3BlZWQKKzE5JSB0byBGaXJlIGFuZCBDb2xkIFJl
c2lzdGFuY2VzIChjcmFmdGVkKQo=

Then you just call:

https://www.poeprices.info/api?l=Delirium&i=

with the encoded data at the end.

If all you had was the min/max and currency from the first result that would be a huge help. But if you could highlight mods that have positive effect that would be extra helpful too.

**Comment by Kyusung4698** [comment]
Closed in favor of #544

**Comment by TheOne320** [comment]
This is off-topic, but wouldn't it be smart to merge duplicate requests instead of just closing them? I am just thinking that some information could be lost.
